src/components/Announcement.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import moment from 'moment-timezone'
+import {
+  Text,
+  View
+} from 'react-native'
+import FontAwesome, { Icons } from 'react-native-fontawesome'
+import { styles } from '../styles'
+
+// const DATE_FORMAT = 'ddd h:mm a';
```

Comment:
Since this code isn't relevant for Announcements you can go ahead and delete it.  In general you want to leave as little commented code in codebase as possible as it later becomes clutter.  If we need it later, we can abstract it out of events.js into it's own utils file and then import it from there.